showcase/app/templates/page-components/application-state.hbs

```diff
@@ -502,7 +502,7 @@
 
   <Shw::Flex @direction="column" @gap="4rem" as |SF|>
     <SF.Item @label="In a card">
-      <Hds::Card::Container @level="mid" @hasBorder={{true}} {{style padding="40px"}}>
+      <Hds::Card::Container @level={{@model.CARD_LEVELS.Mid}} @hasBorder={{true}} {{style padding="40px"}}>
```

Comment:
[question] why for `@level` we need to pass it this way, but for the `@listPosition` above we can pass a plain string?

Reply:
Yup! Added.